Dockerfile.python3

```diff
@@ -0,0 +1,93 @@
+FROM python:3.6-slim-stretch
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+# Allow scripts to detect we're running in our own container
+RUN touch /addons-server-docker-container
+
+# Add nodesource repository and requirements
+ADD docker/nodesource.gpg.key /etc/pki/gpg/GPG-KEY-nodesource
+RUN apt-get update && apt-get install -y \
+        apt-transport-https              \
+        gnupg2                           \
+    && rm -rf /var/lib/apt/lists/*
+RUN cat /etc/pki/gpg/GPG-KEY-nodesource | apt-key add -
+ADD docker/debian-stretch-nodesource-repo /etc/apt/sources.list.d/nodesource.list
+ADD docker/debian-stretch-backports-repo /etc/apt/sources.list.d/backports.list
+
+RUN apt-get update && apt-get install -y \
+        # General (dev-) dependencies
+        bash-completion \
+        build-essential \
+        curl \
+        libjpeg-dev \
+        libsasl2-dev \
+        libxml2-dev \
+        libxslt-dev \
+        locales \
+        zlib1g-dev \
+        libffi-dev \
+        libssl-dev \
+        nodejs \
+        # Git, because we're using git-checkout dependencies
+        git \
+        # Dependencies for mysql-python
+        mysql-client \
+        default-libmysqlclient-dev \
+        swig \
+        gettext \
+        # Use rsvg-convert to render our static theme previews
+        librsvg2-bin \
+        # Use pngcrush to optimize the PNGs uploaded by developers
+        pngcrush \
+        # our makefile and ui-tests require uuid to be installed
+        uuid \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get -t stretch-backports install -y \
+        # For git-based files storage backend
+        libgit2-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+# Compile required locale
+RUN localedef -i en_US -f UTF-8 en_US.UTF-8
+
+# Set the locale. This is mainly so that tests can write non-ascii files to
+# disk.
+ENV LANG en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
+COPY . /code
+WORKDIR /code
+
+ENV PIP_BUILD=/deps/build/
+ENV PIP_CACHE_DIR=/deps/cache/
+ENV PIP_SRC=/deps/src/
+ENV NPM_CONFIG_PREFIX=/deps/
+ENV SWIG_FEATURES="-D__x86_64__"
+
+# Install all python requires
+RUN mkdir -p /deps/{build,cache,src}/ && \
+    ln -s /code/package.json /deps/package.json && \
+    make update_deps && \
+    rm -rf /deps/build/ /deps/cache/
+
+# Preserve bash history across image updates.
+# This works best when you link your local source code
+# as a volume.
+ENV HISTFILE /code/docker/artifacts/bash_history
+
+# Configure bash history.
+ENV HISTSIZE 50000
+ENV HISTIGNORE ls:exit:"cd .."
+
+# This prevents dupes but only in memory for the current session.
+ENV HISTCONTROL erasedups
+
+ENV CLEANCSS_BIN /deps/node_modules/.bin/cleancss
+ENV LESS_BIN /deps/node_modules/.bin/lessc
+ENV UGLIFY_BIN /deps/node_modules/.bin/uglifyjs
+ENV ADDONS_LINTER_BIN /deps/node_modules/.bin/addons-linter
+ENV ADDONS_VALIDATOR_BIN /usr/local/bin/addon-validator
+
+ENV PYTHON_VERSION_MAJOR 3
```

Comment:
should this be further up?  There's a `make` command on line 72

Reply:
54f131d7f7ad6952338ffb44af24b6c4c5e9a577